rhel8/nvidia-driver

```diff
@@ -532,6 +532,23 @@ _install_driver() {
     if [ "${ACCEPT_LICENSE}" = "yes" ]; then
         install_args+=("--accept-license")
     fi
+
+    # Specify the --skip-module-load flag for versions of the nvidia-installer that
+    # support it. From the nvidia-installer help output:
+    #
+    #    --skip-module-load
+    #         Skip the test load of the NVIDIA kernel modules after the modules are built,
+    #         and skip loading them after installation is complete.
+    #
+    # Without this flag, a subtle bug can occur if the nvidia-installer fails to unload
+    # the NVIDIA kernel modules after the test load. The modules will remain loaded and
+    # any custom NVIDIA module parameters configured as input to the driver container
+    # will not be applied.
+    #
+    if [ "${DRIVER_BRANCH}" -ge "570" ]; then
```

Comment:
I know we don't support R550 anymore and R570 is the minimum supported branch with this flag, but I prefer this as it also suggestive of this flag being supported in driver branches starting from R550. Plus, this script would also support building an R550 container image (a non-production one of course) if that is ever needed 
```suggestion
    if [ "${DRIVER_BRANCH}" -ge "550" ]; then
```

Reply:
Thanks!